tests/integration/views/test_dump.py

```diff
@@ -35,16 +35,15 @@ def test_dump_answers_authenticated_with_role_no_answers(self):
 
         # When I haven't submitted any answers
         # And I attempt to dump the answer store
-        self.get('/dump/answers')
+        self.get('/dump/debug')
 
         # Then I get a 200 OK response
         self.assertStatusOK()
 
         # And the JSON response contains an empty array
-        actual = json.loads(self.getResponseData())
-        expected = {'answers': []}
+        response = json.loads(self.getResponseData())
 
-        self.assertDictEqual(actual, expected)
+        self.assertEqual(response['ANSWERS'], [])
```

Comment:
should we also add tests for `METADATA`, `LISTS`, `COMPLETED_BLOCKS` and `COLLECTION_METADATA`?

Reply:
I don't think there should be tests for the specifics of what's returned by the debug endpoint. It now uses the serialise method on questionnaire store, which is well tested elsewhere. I'll remove the tests that are checking for answers in the response.